pkg/controller/autoscaler/horizontalpodautoscaler_controller.go

```diff
@@ -17,21 +17,16 @@ limitations under the License.
 package autoscalercontroller
```

Comment:
This package name is impossible to read. :) Should just be `autoscaler`. Not asking for a change in this PR.